js/vendor/mini-html5-framework/templater.js

```diff
@@ -0,0 +1,31 @@
+function render(template, context) {
+
+    var found = {},
+        re = /{{\s*([^}]+)\s*}}/g,
+        parsedArray = [],
+        curMatch;
+
+    while(curMatch = re.exec(template)) {
+        parsedArray.push(curMatch);
+    }
+
+    parsedArray.forEach(function(match) {
+        for (var prop in context) {
+            found[match[0]] = context[prop];
+        }
+    });
+
+    function parser(template) {
```

Comment:
Just wondering, why did you choose to use a nested function here?
I'm slightly averse to these unless required for specific reasons (e.g. closures or recursion). Is it to make it private?

Generally I'd prefer making it a regular function and passing in `found` and `context` explicitly as parameters, to make the data flow clearer.